appcelerator-archive/amp
# Kibana index and dashboard auto creation

- creation of index pattern ampbeat-*
- creation of search, visualization, dashboard objects

how to test:
deploy the monitoring stack, give time to elasticsearch and kibana services to be up, and check the dashboards on the kibana UI (localhost:5601)
